src/utils/cs.ts

```diff
@@ -110,17 +109,16 @@ const cs = (function() {
       input.style.background = '#fff7d6';
       input.style.borderColor = '#f4dc94';
     }
-  }
+  };
 
   /**
    * Filters Select to only return children with <option> tagName
    * @param {NodeList} selectList
    */
-  const filterSelectList = (selectList: NodeList): Array<Object> => {
-    return Array.from(selectList).filter((op: HTMLOptionElement) => {
-      return op.tagName === 'OPTION'; // push only <option>
-    });
-  }
+  const filterSelectList = (selectList: NodeList): Array<Object> =>
+    Array.from(selectList).filter(
```

Comment:
Should this be `return Array.from(selectedList).filter(...)`?

Reply:
I didn't make any changes to logic. These are all eslint autofixes

Reply:
The infra wasn't configured to run against `.ts` files. Only `.tsx` files. So many files were not linted

Reply:
Ahh, I totally missed that it's just returning it (no curly brace). Makes sense!